app/views/assessments/_edit_problems.html.erb

```diff
@@ -1,7 +1,35 @@
-<script type="text/javascript">
-  is_positive_grading = <%= @is_positive_grading %>;
-  has_annotations = <%= @has_annotations %>;
-</script>
+<% content_for :javascripts do %>
+    <script type="text/javascript">
+        is_positive_grading = <%= @is_positive_grading %>;
+        has_annotations = <%= @has_annotations %>;
+
+        document.addEventListener('DOMContentLoaded', function() {
```

Comment:
Nit: for consistency with other files, can we use `$document.ready( ... )` instead?